CMakeLists.txt

```diff
@@ -217,6 +219,31 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/zenoh-pico
   COMPONENT Headers
 )
 
+set(CMAKE_INSTALL_CMAKEDIR "/usr/lib/cmake/zenohpico")
```

Comment:
It's better to use ${CMAKE_INSTALL_LIBDIR} instead of "/usr/lib"

Reply:
It shall be fixed now.